imagebuildah/executor.go

```diff
@@ -381,6 +386,8 @@ func (b *Executor) getImageTypeAndHistoryAndDiffIDs(ctx context.Context, imageID
 		diffIDs:      oci.RootFS.DiffIDs,
 		err:          nil,
 	}
+	logrus.Debugf("updated image %s in cache:", imageID)
+	util.LogJson(imageID, b.imageInfoCache[imageID])
```

Comment:
actually, this also provides an empty {} result for ``b.imageInfoCache[imageID]``
I'm probably doing it wrong!

Reply:
Is it empty if you pass it to Debugf instead?

Reply:
it wasn't. fixed that now